src/Silex/Application/FormTrait.php

```diff
@@ -25,11 +25,12 @@
      *
      * @param mixed $data    The initial data for the form
      * @param array $options Options for the form
+     * @param mixed $type    Type of the form
```

Comment:
why ``mixed`` ? The only supported types are ``FormTypeInterface`` and ``string``